frontend-app/src/yash_ui_task/Components/student.jsx

```diff
@@ -0,0 +1,64 @@
+import React from "react";
+import '../../App.css';
+import { useState } from "react";
+import { data } from "../Data";
+import {columns} from "../Columns";
+import { Table,Row,Col,Card,Button } from "antd";
+import { Input} from 'antd';
+const {Search} = Input;
+
+
+export default function Student(){
+
+    const [studentData,setStudentData]= useState(data);
+    const [columnsHeader,setColumnsHeaders]= useState(columns);
+
+    const onSearch= value => setStudentData(
+        value===""?data:data.filter(entry =>
+        entry.name.includes(value)||
+        entry.passoutYear.includes(value)||
+        entry.percent.includes(value)||
+        entry.city.includes(value)||
+        entry.id.includes(value)
+      ));
+
+    const onRefresh=()=>{
+        setTimeout(() => {setStudentData(data)}, 3000);
+    }  
+    return(
+        <div>
+            <Card className="boxStyle">
```

Comment:
bottom border of table should be black in color.

Reply:
done